docs/advanced-ai/intro-tutorial.md

```diff
@@ -1,217 +1,126 @@
 ---
 #https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
-description: Learn how to build AI workflows using n8n's LangChain implementation. You'll build a chat workflow that uses ChatGPT and pulls in data from other n8n workflows.
+description: Learn how to build AI workflows with n8n
 type: tutorial
 ---
 
-# Tutorial: Build an AI workflow in n8n
+# Tutorial: Build an AI chat agent with n8n
 
-This tutorial introduces LangChain functionality in n8n. You can work through it with no prior knowledge of LangChain, AI, or n8n. However, if you've never used n8n before, you should also do the [Longer quickstart](/try-it-out/longer-introduction/), which introduces key n8n concepts.
+- TODO: Introduction text
 
-!["Screenshot of the workflow"](/_images/advanced-ai/ai-tutorial-outcome.png)
+!["Screenshot of the completed workflow"](/_images/advanced-ai/ai-intro01.png)
 
-In this tutorial you will:
 
-* Create a workflow from scratch. It uses the Chat Trigger to simulate and test chat interactions, ChatGPT to power the chat functionality, and a custom tool to connect to other n8n workflows.
-* Understand key concepts, including:
-	* The role of agents, models, and tools when building AI functionality.
-	* Cluster nodes in n8n: groups of root and sub-node.
-	* Loading your own data into AI workflows.
+Many people find it easier to take in new information in video format. This tutorial is based on one of n8n's popular videos, linked below. Watch the video or read the steps here, or both!
 
-## Step one: Sign up for n8n
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yzvLfHb0nqE?si=7ruaUEycFcoQbYsD" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 
---8<-- "_snippets/try-it-out/install-run-n8n.md"
+### What you will need
 
-/// note | n8n version 1.21.0 or above
-This tutorial requires n8n version 1.21.0 or above.
-///
+- n8n. For this tutorial we recommend using the [n8n cloud](/choose-n8n/cloud/) service - there is a free trial for new users!. For a self hosted service, refer to the [installation pages](/hosting/installation/docker/).
```

Comment:
For prerequisites like this, I think it's a bit easier to scan if we use a `- **requirement**: explanation text` format.

Reply:
yup. this was I think because i just added everything as bullets to begin with